keyring/backends/libsecret.py

```diff
@@ -0,0 +1,103 @@
+from contextlib import closing
+import logging
+
+from ..util import properties
+from ..backend import KeyringBackend
+from ..credentials import SimpleCredential
+from ..errors import (
+    PasswordDeleteError,
+    ExceptionRaisedContext,
+    KeyringLocked,
+)
+
+available = False
+try:
+    import gi
+    gi.require_version('Secret', '1')
+    from gi.repository import Secret
+    available = True
+except ImportError:
```

Comment:
I think the `require_version` call can also raise ValueError.

```suggestion
except (ImportError, ValueError):
```